certification/formatters/formatters.go

```diff
@@ -1,95 +1,89 @@
+// passedPolicies := make([]policy.Metadata, len(r.Passed))
+// failedPolicies := make([]policy.Policy, len(r.Failed))
+// erroredPolicies := make([]policy.Policy, len(r.Errors))
+
+// if len(r.Passed) > 0 {
+// 	for i, policyData := range r.Passed {
+// 		passedPolicies[i] = policyData.GetMetadata()
+// 	}
+// }
+
+// if len(r.Failed) > 0 {
+// 	for i, policyData := range r.Failed {
+// 		failedPolicies[i] = policy.Policy{
+// 			Metadata: policyData.GetMetadata(),
+// 			HelpText: policyData.GetHelp(),
+// 		}
+// 	}
+// }
+
+// if len(r.Errors) > 0 {
+// 	for i, policyData := range r.Errors {
+// 		erroredPolicies[i] = policyData.GetHelp()
+// 	}
+// }
+
+// response := runtime.UserResponse{
+// 	Image:             r.TestedImage,
+// 	ValidationVersion: version.Version,
+// 	Results: runtime.UserResponseText{
+// 		Passed: passedPolicies,
+// 		Failed: failedPolicies,
+// 		Errors: erroredPolicies,
+// 	},
+// }
+
+// return response
+
+// }
 
 // GenericJSONFormatter is a FormatterFunc that formats results as JSON
-func GenericJSONFormatter(r runtime.Results) ([]byte, error) {
-	response := GetResponse(r)
-
-	responseJSON, err := json.MarshalIndent(response, "", "    ")
-	if err != nil {
-		e := fmt.Errorf("%w with formatter %s: %s",
-			errors.ErrFormattingResults,
-			"json",
-			err,
-		)
-
-		return nil, e
-	}
-
-	return responseJSON, nil
-}
-
-// GenericXMLFormatter is a FormatterFunc that formats results as XML
-func GenericXMLFormatter(r runtime.Results) ([]byte, error) {
-	response := GetResponse(r)
-
-	responseJSON, err := xml.MarshalIndent(response, "", "    ")
-	if err != nil {
-		e := fmt.Errorf("%w with formatter %s: %s",
-			errors.ErrFormattingResults,
-			"json",
-			err,
-		)
-
-		return nil, e
-	}
-
-	return responseJSON, nil
-}
-
-func JUnitXMLFormatter(r runtime.Results) ([]byte, error) {
-	return nil, fmt.Errorf("%w: The JUnit XML Formatter is not implemented", errors.ErrFeatureNotImplemented)
-}
+// func GenericJSONFormatter(r runtime.Results) ([]byte, error) {
+// 	response := GetResponse(r)
+
+// 	responseJSON, err := json.MarshalIndent(response, "", "    ")
+// 	if err != nil {
+// 		e := fmt.Errorf("%w with formatter %s: %s",
+// 			errors.ErrFormattingResults,
+// 			"json",
+// 			err,
+// 		)
+
+// 		return nil, e
+// 	}
+
+// 	return responseJSON, nil
+// }
+
+// // GenericXMLFormatter is a FormatterFunc that formats results as XML
+// func GenericXMLFormatter(r runtime.Results) ([]byte, error) {
+// 	response := GetResponse(r)
+
+// 	responseJSON, err := xml.MarshalIndent(response, "", "    ")
+// 	if err != nil {
+// 		e := fmt.Errorf("%w with formatter %s: %s",
+// 			errors.ErrFormattingResults,
+// 			"json",
+// 			err,
+// 		)
+
+// 		return nil, e
+// 	}
+
+// 	return responseJSON, nil
+// }
+
+// func JUnitXMLFormatter(r runtime.Results) ([]byte, error) {
+// 	return nil, fmt.Errorf("%w: The JUnit XML Formatter is not implemented", errors.ErrFeatureNotImplemented)
+// }
```

Comment:
Just remove the code. We don't want to commit large blocks of commented code.

Reply:
@bcrochet Actually, we need to go the other way. All of the output handlers were removed in this PR, and we need to rewire those in.

Reply:
That's fine. Just don't want the final commit to contain large blocks of commented code.